src/scenes/FAQ/FAQ.js

```diff
@@ -29,29 +28,29 @@ export default class Faq extends React.Component {
 
   renderHeader = (section, index) => {
     const bounce = this.state[`arrowBounce${index}`];
+    const animationConfig = {
+      transform: [{
+        rotate: bounce.interpolate({
+          inputRange: [0, 360],
+          outputRange: ['0deg', '360deg']
+        })
```

Comment:
When you finally that asshole at work to upgrade your dependencies, you can use the native driver flag in animations to help with performance boosts.

`useNativeDriver: true`

http://facebook.github.io/react-native/releases/next/docs/animated.html#using-the-native-driver